modules/apps/29-fee/client/cli/cli.go

```diff
@@ -18,6 +18,7 @@ func GetQueryCmd() *cobra.Command {
 		GetCmdTotalRecvFees(),
 		GetCmdTotalAckFees(),
 		GetCmdTotalTimeoutFees(),
+		GetIncentivizedPacketByChannel(),
```

Comment:
I think we should use `GetCmd...` and align with grpc query naming

```suggestion
		GetCmdIncentivizedPacketForChannel(),
```